### ISSUE TITLE
Specific site not reloading

### ISSUE THREAD

**Issue by saladgoat** [nt1m/livemarks#353]
On one of the many sites I follow, it never reloads the live bookmark. Even when I click Reload Live Bookmark, nothing happens. If I close Firefox (up-to-date) and re-open it, the Live Bookmarks are all new for that site, but it will never refresh while FF is open.
The site is https://3downnation.com/ - hopefully you can investigate and fix it, or advise what I am doing wrong.
Thanks!

**Comment by saladgoat** [comment]
So then this happened today - the main title has a checkmark, as if all items are read, but as you can see _none_ of them are read.
![Image2](https://user-images.githubusercontent.com/7467557/116647371-3dc7e680-a92f-11eb-9969-23e5060c10b4.png)
At least I was able to use Open All Unread Pages. However, after doing so, the world icons were replaced by the site's favicon, but no checkmarks on the read pages. 
Again, this is the only site this is happening on. And Firefox is up to date.

**Comment by evilpie** [comment]
I can't reproduce this issue. After clicking on one of the bookmarks the favicon and checkmark appears.

**Comment by saladgoat** [comment]
The second issue cleared up pretty fast, within a day or two. But the original issue persists - the list will not reload, even when clicking Reload Live Bookmark. It is still the only one that does not work. All others reload on their own, except for one, which requires that I click the reload option, and then it does reload. But this particular site will not reload at any interval or on request. I did notice it reloaded itself yesterday or this morning, so maybe it will do it once a week, but not on demand.

**Comment by evilpie** [comment]
I don't really have time to wait for the feed to update. What I tried was to remove some the livemark feed entries and then clicking "Reload Live Bookmark". That seemed to at least recreated the deleted ones.

**Comment by saladgoat** [comment]
@evilpie Yes, I can confirm it seems to be loading new items on its own at regular intervals. Haven't tried reloading because it's always new items, so I guess you can mark this one as solved. Thanks!
